src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.cs

```diff
@@ -238,6 +420,28 @@ public static Vector4 Out(Vector4 destination, Vector4 source)
         return color;
     }
 
+    /// <summary>
+    /// Returns the result of the "Out" compositing equation.
+    /// </summary>
+    /// <param name="destination">The destination vector.</param>
+    /// <param name="source">The source vector.</param>
+    /// <returns>The <see cref="Vector256{Single}"/>.</returns>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static Vector256<float> Out(Vector256<float> destination, Vector256<float> source)
+    {
+        // calculate alpha
+        Vector256<float> sW = Avx.Shuffle(source, source, ShuffleAlphaControl);
+        Vector256<float> dW = Avx.Shuffle(destination, destination, ShuffleAlphaControl);
+        Vector256<float> alpha = Avx.Multiply(Avx.Subtract(Vector256.Create(1F), dW), sW);
```

Comment:
Thought I saw Tanner comment on this yesterday, but it's disappeared.  You can save an instruction by doing the alpha shuffle after the multiply.

Reply:
My comment should be here, hopefully it hasn't actually disappeared https://github.com/SixLabors/ImageSharp/pull/2359#discussion_r1111157263 😅

Reply:
Ha, yeah, it just got marked as outdated so it doesn't show up in the code view anymore.  Saw it after I submitted the comments :)

Reply:
I've implemented that now.